### ISSUE TITLE
What to do after a PITR?

### ISSUE THREAD

**Issue by renard** [EnterpriseDB/barman#218]
Barman version: 2.5
Postgresql client:  9.6.10
OS: Debian 9.5 

For some reason I had to perform a PITR. I used following command:

```
 barman recover --remote-ssh-command 'ssh PG_HOST' --target-action=promote --target-time '2019-04-15 10:40' PG_BACKUP 20190414T201001 /srv/postgresql/lib/9.6/recover/
```

The restoration worked fine and the PG_HOST is OK.

Since then barman was in failure:

```
2019-04-15 13:05:01,912 [1602] barman.wal_archiver INFO: Activating WAL archiving through streaming protocol
2019-04-15 13:05:01,980 [1602] barman.command_wrappers INFO: PG_BACKUP : pg_receivexlog: starting log streaming at 2/42000000 (timeline 1)
2019-04-15 13:05:01,982 [1602] barman.command_wrappers INFO: PG_BACKUP : pg_receivexlog: could not send replication command "START_REPLICATION": ERROR:  requested starting point 2/42000000 on tim
eline 1 is not in this server's history
2019-04-15 13:05:01,984 [1602] barman.command_wrappers INFO: PG_BACKUP : DETAIL:  This server's history forked from timeline 1 at 2/4043B728.
2019-04-15 13:05:01,986 [1602] barman.command_wrappers INFO: PG_BACKUP : pg_receivexlog: disconnected
2019-04-15 13:05:01,987 [1602] barman.server ERROR: ArchiverFailure:pg_receivexlog terminated with error code: 1
```

I didn't see anywhere in the documentation what to do once the recover has been done.

Having a closer look shows that `streaming` repository has a partial WAL:
`000000010000000200000042.partial`.

I just removed the `streaming` repository and everything was OK on both barman and postgresql.

So what is the correct procedure to follow once a server has been recovered using a PITR?

**Comment by frutality** [comment]
Would love to read about it too. Just met exactly same issue:
1. Configured backups
2. Simulated disaster
3. Made a recover (all good)

After that, replication stopped working until I deleted .partial file in my /var/lib/barman/test/streaming/ directory. Completely not sure if I did everything right. Kind of afraid deploy to production without being certain.

@renard have you learned something useful about it for last months?
